src/libexpr/eval.cc

```diff
@@ -2632,9 +2634,11 @@ void EvalState::maybePrintStats()
 
 void EvalState::printStatistics()
 {
+#ifndef _WIN32 // TODO use portable implementation
```

Comment:
https://github.com/postgres/postgres/blob/master/src/port/win32getrusage.c can help with this

(Create issue?)

Reply:
#10542